docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Genesis Launcher - Running'
+sidebar_label: 'Running'
+id: running-launcher
+keywords: [server, tooling, code snippets, launcher, jetbrains, local]
+tags:
+  - server
+  - tooling
+  - code snippets
+  - launcher
+---
+
+## Starting the Launcher
+
+After syncing your gradle project, there should now be a `genesisLauncher` gradle task.
+You can start the launcher by running that task from an IDE, or from the command line, run:
+
+```shell
+./gradlew genesisLauncher
+```
+
+## Usage
+
+Once you open Genesis Launcher for your project, all the services will be listed, including the Web UI.
+To get started for the first time; click on the 🚀icon, just underneath the Genesis logo.
+
+This will bootstrap the database and launch the application.
+Once the database has been initialised, feel free to run "Start All" instead.
+
+Bootstrap will run "Start All" as well, but will first run [remap](/operations/commands/remap/) and ConsolidateRights.
+
+Prior to starting service, build tasks will be run.
+
```

Comment:
We should add an explicit section around compact processes and enabling it to save local PC resource utilisation (as this comes up a lot with clients)

![image](https://github.com/user-attachments/assets/e1b9ffb7-87e2-4c6a-8981-b463fd891f37)